packages/common/src/scopeSupportFacets/scopeSupportFacets.types.ts

```diff
@@ -32,12 +32,16 @@ const scopeSupportFacets = [
   "namedFunction",
   "namedFunction.method",
   "namedFunction.method.iteration.class",
+  "namedFunction.constructor",
+  "namedFunction.constructor.iteration.class",
```

Comment:
This seems redundant. What's the difference between this and the method iteration class one? They're both just a class

Reply:
It's a different node in the parse tree so you need to define it separately and I kind of wanted us to force them to have that scope test?

Reply:
done